azure_pipeline.yaml

```diff
@@ -123,6 +131,7 @@ parameters:
       storage_account_rg: 'core-infra-ithc-rg'
       storage_account_name: 'cftappsithc'
       dependsOn: 'sbox_private_dns'
+      pipeline_tests: true
```

Comment:
i'm not sure about this, unlikely dns resolution would break by the time the test runs and this really shouldn't be a component anyway it should just be in the appgw component